tests/Main.hs

```diff
@@ -0,0 +1,48 @@
+module Main
+  ( main
+  )
+where
+
+import Prelude
+
+import Control.Monad (void)
+import Test.HSpec.JUnit (junitFormat)
+import Test.Hspec
+import Test.Hspec.Runner
+
+main :: IO ()
+main = do
+  summary <- flip runSpec defaultConfig $ do
+    describe "XML output" $ do
+      it "matches golen file" $ do
+        -- pendingWith
+        --   "Need to normalize time and timestamp values to match golden"
+        makeJunitResults
+
+        golden <- readFile "tests/golden.xml"
+        test <- readFile ".temp/test.xml"
```

Comment:
:+1: I'm into this sort of testing for this library.

Reply:
Yeah, this is really cool